array-methods-basic/basic-map.js

```diff
@@ -3,12 +3,18 @@
 2. Write more tests to check it is working properly
 */
 
+const doubleAll = (numbers)=>{
```

Comment:
Same error like in other files. Don't use .push, just return the value.
Remember that .map returns a new array.
```javascript
const doubleAll = (numbers) => {
  const result = numbers.map((number) => {
    return number*2;
    });
  return result
}
```